conf/base.config

```diff
@@ -37,8 +37,8 @@ process {
     time = { check_max( 6.h * task.attempt, 'time' ) }
   }
   withLabel:process_medium {
-    cpus = { check_max( 8 * task.attempt, 'cpus' ) }
-    memory = { check_max( 32.GB * task.attempt, 'memory' ) }
+    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
```

Comment:
This should be either reverted or all the labels adapted (such that low is actually lower than medium). Or add more labels that tweak time only, if that is what you wanted to do.